app/src/main/kotlin/com/saltedge/authenticator/features/connections/select/SelectConnectionsFragment.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * This file is part of the Salt Edge Authenticator distribution
+ * (https://github.com/saltedge/sca-authenticator-android).
+ * Copyright (c) 2020 Salt Edge Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 3 or later.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * For the additional permissions granted for Salt Edge Authenticator
+ * under Section 7 of the GNU General Public License see THIRD_PARTY_NOTICES.md
+ */
+package com.saltedge.authenticator.features.connections.select
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.saltedge.authenticator.R
+import com.saltedge.authenticator.features.connections.common.ConnectionViewModel
+import com.saltedge.authenticator.features.connections.list.ConnectionsListAdapter
+import com.saltedge.authenticator.interfaces.ListItemClickListener
+import com.saltedge.authenticator.interfaces.UpActionImageListener
+import com.saltedge.authenticator.tool.ResId
+import com.saltedge.authenticator.tool.log
+import com.saltedge.authenticator.widget.fragment.BaseFragment
+import com.saltedge.authenticator.widget.list.SpaceItemDecoration
+import kotlinx.android.synthetic.main.fragment_select_connections.*
+
+const val KEY_CONNECTIONS = "CONNECTIONS"
+
+class SelectConnectionsFragment : BaseFragment(), ListItemClickListener, UpActionImageListener {
+
+    private val adapter = ConnectionsListAdapter(clickListener = this)
+    private var headerDecorator: SpaceItemDecoration? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setHasOptionsMenu(false)
+        adapter.data = arguments?.getSerializable(KEY_CONNECTIONS) as List<ConnectionViewModel>
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        activityComponents?.updateAppbarTitleWithFabAction(getString(R.string.choose_connections_feature_title))
```

Comment:
do you have multiple selection or single selection mode? 
accordingly fix the title resource.